### ISSUE TITLE
Packaged Mac build of pong example crashes on startup

### ISSUE THREAD

**Issue by garyo** [touilleMan/godot-python#58]
I built godot-python on Mac using cypthon backend, like this:
`scons platform=osx-64 backend=cpython godot_binary=/Applications/Godot.app/Contents/MacOS/Godot  gdnative_parse_cpp="clang -E"`

If I package the python pong example as a DMG (from the editor) and run that, it crashes on startup. It works fine when run from Godot though.

```
% lldb /Volumes/pypong/pypong.app/Contents/MacOS/pypong         
(lldb) target create "/Volumes/pypong/pypong.app/Contents/MacOS/pypong"
Current executable set to '/Volumes/pypong/pypong.app/Contents/MacOS/pypong' (x86_64).
(lldb) run
Process 79358 launched: '/Volumes/pypong/pypong.app/Contents/MacOS/pypong' (x86_64)
arguments
0: /Volumes/pypong/pypong.app/Contents/MacOS/pypong
Current path: /Volumes/pypong/pypong.app/Contents/Resources
2018-02-28 14:52:11.345460-0500 pypong[79358:3128294] MessageTracer: Falling back to default whitelist
OpenGL ES 3.0 Renderer: Intel Iris OpenGL Engine
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fff99a9f340 (most recent call first):
Process 79358 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
    frame #0: 0x00007fff60b98e3e libsystem_kernel.dylib`__pthread_kill + 10
libsystem_kernel.dylib`__pthread_kill:
->  0x7fff60b98e3e <+10>: jae    0x7fff60b98e48            ; <+20>
    0x7fff60b98e40 <+12>: movq   %rax, %rdi
    0x7fff60b98e43 <+15>: jmp    0x7fff60b900b8            ; cerror_nocancel
    0x7fff60b98e48 <+20>: retq   
Target 0: (pypong) stopped.
(lldb) bt
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
  * frame #0: 0x00007fff60b98e3e libsystem_kernel.dylib`__pthread_kill + 10
    frame #1: 0x00007fff60cd7150 libsystem_pthread.dylib`pthread_kill + 333
    frame #2: 0x00007fff60af5312 libsystem_c.dylib`abort + 127
    frame #3: 0x0000000113c131b1 libpython3.6m.dylib`Py_FatalError(msg=<unavailable>) at pylifecycle.c:1466 [opt]
    frame #4: 0x0000000113c12f94 libpython3.6m.dylib`_Py_InitializeEx_Private(install_sigs=0, install_importlib=1) at pylifecycle.c:444 [opt]
    frame #5: 0x0000000113a02f7b libpythonscript.dylib`_cffi_py_initialize + 11
    frame #6: 0x0000000113a02870 libpythonscript.dylib`_cffi_carefully_make_gil + 288
    frame #7: 0x0000000113a0268d libpythonscript.dylib`_cffi_start_python + 13
    frame #8: 0x0000000113a025ff libpythonscript.dylib`_cffi_start_and_call_python + 31
    frame #9: 0x0000000113a018f2 libpythonscript.dylib`pybind_init + 50
    frame #10: 0x000000010017daec pypong`PluginScriptLanguage::init() + 12
    frame #11: 0x0000000101457e79 pypong`ScriptServer::init_languages() + 41
    frame #12: 0x0000000100027282 pypong`Main::setup2(unsigned long long) + 4594
    frame #13: 0x0000000100023dca pypong`Main::setup(char const*, int, char**, bool) + 18058
    frame #14: 0x000000010000e422 pypong`main + 626
    frame #15: 0x00007fff60a49115 libdyld.dylib`start + 1
    frame #16: 0x00007fff60a49115 libdyld.dylib`start + 1
(lldb) 
```

It may be relevant that my Python is python3.6, built in my home dir. That's in my path, not MacOS system python. But as I understand, godot-python is supposed to be self-contained.

**Comment by garyo** [comment]
If I point to my python3 lib dir with PYTHONPATH, then it works:

`PYTHONPATH=/Users/garyo/python36/lib/python3.6 /Volumes/pypong/pypong.app/Contents/MacOS/pypong`

**Comment by touilleMan** [comment]
fixed on v0.40.0
